horizon/factdb/client.py

```diff
@@ -48,13 +57,14 @@ def _configure_api_routes(self, app: FastAPI):
         policy_router = init_policy_router(policy_updater=self.policy_updater)
         data_router = init_data_router(data_updater=self.data_updater)
         policy_store_router = init_policy_store_router(authenticator)
-        callbacks_router = init_callbacks_api(authenticator, self._callbacks_register)
 
         # mount the api routes on the app object
         app.include_router(policy_router, tags=["Policy Updater"])
         app.include_router(data_router, tags=["Data Updater"])
         app.include_router(policy_store_router, tags=["Policy Store"])
-        app.include_router(callbacks_router, tags=["Callbacks"])
```

Comment:
you removed the callback router initialization and inlcude but you didn't add it anywhere else, why ?

Reply:
We do not allow direct configuration of OPAL Callbacks, that's why I've added this PDP passthrough.

I've removed them since they are inaccessible (require JWT authentication that the clients does not have) and can break things by altering our internal callbacks.